Replaced unsupported Medium attribute on methods with Group('slow')

maryo opened the pull request:
`#[Medium]` attribute can target only classes, `#[Group('medium')]` also cannot be applied.
`Group name "medium" is not allowed for method tests\integration\Cache\Psr6CacheTest::testExpiration`
Therefore i used `#[Group('slow')]`, or it can be dropped entirely, or a different name.

Nyholm commented:
Oh no.. I merged #119. I did not see this. Im sorry. 

Is there anything left to do if you rebase on master?

maryo commented:
Yes, rebased, there is still the issue with Medium attribute.
```
An error occurred inside PHPUnit.

Message:  Attribute "PHPUnit\Framework\Attributes\Medium" cannot target method (allowed targets: class)
```

`@medium will be implemented as PHPUnit\Framework\Attributes\Medium (only allowed on classes; @medium was allowed on methods but it does not make sense to mix tests of different sizes in a test case class)`
https://github.com/sebastianbergmann/phpunit/issues/4502

I replaced it with `Group('slow')` (there is `sleep(3)`), or it can be dropped entirely, or do you prefer a different group name?

Nyholm commented:
Thank you. I think it looks alright. 

@xabbuh can you please review this?

xabbuh commented:
👍 works for me

Nyholm commented:
Thank you
